src/app/Instance/BridgeOverview/Components/BOTableView.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import SEStatusLabel from "@app/components/SEStatusLabel/SEStatusLabel";
+import {
+  Button,
+  Divider,
+  Flex,
+  FlexItem,
+  Label,
+  Split,
+  SplitItem,
+  Stack,
+  StackItem,
+  Text,
+  TextContent,
+  Truncate,
+} from "@patternfly/react-core";
+import {
+  ActionsColumn,
+  TableComposable,
+  Tbody,
+  Td,
+  Tr,
+} from "@patternfly/react-table";
+import { ManagedResourceStatus } from "@rhoas/smart-events-management-sdk";
+import { DemoData } from "../BridgeOverview";
+
+interface BOTableViewProps {
+  demoData: DemoData[];
+  name: string;
+}
+
+export const BOTableView = (props: BOTableViewProps): JSX.Element => {
+  const rowActions = [{ title: "Edit" }, { title: "Delete" }];
+
+  const { demoData, name } = props;
+
+  return (
+    <>
+      <Split hasGutter>
+        <SplitItem isFilled>
+          <TextContent>
+            <Text component="h4">{name}</Text>
+          </TextContent>
+        </SplitItem>
+        <SplitItem>
+          <Button variant="secondary">Create {name}</Button>
+        </SplitItem>
+      </Split>
+      <Divider style={{ paddingTop: "1rem" }} />
```

Comment:
You could move the inline style to a separate CSS file for this component. I would also use [a PF spacer](https://www.patternfly.org/v4/guidelines/spacers) as value here.

I already added a CSS file for `BOTable` [in my previous PR](https://github.com/ppadti/sandbox-ui/pull/5/files#diff-5838c8e7ea9d255edbb9a634c2cc36af89690dc5263b682af55df20453bc32b2) but it looks gone now.